tests/smtp-raw-extraction/test.yaml

```diff
@@ -0,0 +1,10 @@
+requires:
+  features:
+    - HAVE_NSS
+  files:
+    - src/output-filestore.c
+
```

Comment:
think we need a 'min-version: 5.0.0' here in the requires, otherwise this will break for 4.0 and 4.1